shoop/utils/text.py

```diff
@@ -28,8 +30,8 @@ def flatten(str, whitespace="-"):
     Flatten the given text into lowercase ASCII, removing diacriticals etc.
     Replace runs of whitespace with the given whitespace replacement.
 
-    >>> flatten(u"hellö, wörld")
-    "hello,-world"
+    >>> print(flatten("hellö, wörld"))
```

Comment:
But the repr will be `u'hello,-world'` for Py2 and `'hello,-world'` for Py3. That's why I added the print.